java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -155,25 +155,30 @@ class ReferenceModeStore private constructor(
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     val backingStore = DirectStoreMuxer<CrdtData, CrdtOperation, Any?>(
         storageKey = backingKey,
-        backingType = backingType,
-        callbackFactory = { muxId ->
-            ProxyCallback { message ->
-                CoroutineScope(coroutineContext).launch {
-                    receiveQueue.enqueue(
-                        PreEnqueuedFromBackingStore(message.toReferenceModeMessage(), muxId)
-                    )
-                }
-            }
-        }
+        backingType = backingType
     )
 
+    val backingStoreId: Int
```

Comment:
Does this need to be public?

Reply:
I'd like to be able to use it in tests. Is there another way to do this without using `VisibleForTesting`?